setup.py

```diff
@@ -134,6 +134,9 @@ def make_pybind11_extension(
     rpathflag = '-Wl,-rpath,%s/lib' % sys.exec_prefix
     if "CONDA_PREFIX" in os.environ:
         include_dirs.append(os.path.join(os.environ["CONDA_PREFIX"], "include"))
+    else:
+        include_dirs.append(os.path.join("/usr/include/scotch"))
+        include_dirs.append(os.path.join("/usr/include/metis"))
```

Comment:
SOLVCON C++ part now uses exclusive SCOTCH.  We may not need Metis at this point.